hibernation-setup-tool.c

```diff
@@ -824,6 +824,28 @@ static void perform_fs_specific_checks(const char *path)
     }
 }
 
+bool check_if_system_version_is_higher(const char *version) {
```

Comment:
If something is supported in `4.18`, this should be `is_kernel_version_at_least()` and compare using `>=` instead of `>`, right? It's a lot easier to support this (e.g. if you read `Feature XXXXX is available since kernel 5.10` in a man page).